frontend/src/components/template/TheNavBar.vue

```diff
@@ -31,6 +31,7 @@
           <div class="dropdown">
             <span>Coverage Maintenance</span>
             <div class="dropdown-content">
+              <router-link @click="changeEffectiveDate" :class="menuClass($route, 'ChangeEffectiveDate')" :to="{ name: 'ChangeEffectiveDate' }" v-if="hasPermission('ChangeEffectiveDate')">Change Effective Date</router-link>
```

Comment:
@click should call resetAlert. changeEffectiveDate is throwing an error in the console.